tests/unit/tags.t

```diff
@@ -622,6 +622,41 @@ is_deeply(
 	}
 ) or diag explain $tag_ref;
 
+# test remove_tags_from_fields
+
+# removing 2 tags in a list of 4
+$product_ref = {
+	lc => "fr",
+	categories => "pommes, bananes, en:pears, fr:fraises, es:limones",
+};
+
+remove_tag_from_field($product_ref, "fr", "categories", "bananes, fraises");
```

Comment:
you forgot to change the name here.

Better run your tests locally before committing:
* make test-unit test=tags.t

(and make test-int test=youfile.t for the integration)
See https://openfoodfacts.github.io/openfoodfacts-server/dev/how-to-write-and-run-tests/